recipes/coin-lemon/all/conandata.yml

```diff
@@ -6,3 +6,9 @@ patches:
   "1.3.1":
     - patch_file: "patches/cmake-add-runtime-destination_1.3.1.patch"
     - patch_file: "patches/0001-remove-register-keyword.patch"
+    - patch_file: "patches/fix_cpp_17_compilation_warnings.patch"
+      patch_type: "portability"
+      patch_description: "Fix C++17 compilation warnings regarding the use of the deprecated std::iterator"
+    - patch_file: "patches/fix_cpp_20_compilation_errors.patch"
+      patch_type: "portability"
+      patch_description: "Fix C++20 compilation errors regarding the use of allocator.construct and allocator.destroy"
```

Comment:
Please add the `patch_source` too with the PR to upstream were you submitted this 🙏 so others can benefit from it too

Reply:
Hi @prince-chrismc ,

I tried to get in touch with the maintainers of the Lemon Graph library via their [website](https://lemon.cs.elte.hu/trac/lemon) and their user [mailing list](https://lemon.cs.elte.hu/trac/lemon/wiki/MailingLists), without luck. The last version of the library is from 2014.

I've issued a new ticket, and attached the first of the patches (C++17), to: https://lemon.cs.elte.hu/trac/lemon/attachment/ticket/680/fix_cpp_17_compilation_warnings.patch
The second patch (C++20) is actually part of this other attachment to a previous ticket (I only took the patch section for `array_map.h` because the patch section for `path.h` wasn't patching correctly): http://lemon.cs.elte.hu/trac/lemon/attachment/ticket/631/93c983122692.patch

I think I should update this second patch with the fixes for `path.h` as well.

If you reckon, I can add those two links mentioned above as `patch_source`.

Thanks,
Roberto.